src/components/pages/rules/rulesGrid/rulesGrid.js

```diff
@@ -38,16 +39,21 @@ export class RulesGrid extends Component {
 
     this.contextBtns = {
       disable:
-      <Btn key="disable" className="rule-status-btn" svg={svgs.disableToggle} onClick={this.openStatusFlyout}>
-        <Trans i18nKey="rules.flyouts.disable">Disable</Trans>
-      </Btn>,
-      enable: <Btn key="enable" className="rule-status-btn enabled" svg={svgs.enableToggle} onClick={this.openStatusFlyout}>
-        <Trans i18nKey="rules.flyouts.enable">Enable</Trans>
-      </Btn>,
-      changeStatus: <Btn key="changeStatus" className="rule-status-btn" svg={svgs.changeStatus} onClick={this.openStatusFlyout}>
-        <Trans i18nKey="rules.flyouts.changeStatus">Change status</Trans>
-      </Btn>,
-      edit: <Btn key="edit" svg={svgs.edit} onClick={this.openEditRuleFlyout}>{props.t('rules.flyouts.edit')}</Btn>
+        <Btn key="disable" className="rule-status-btn" svg={svgs.disableToggle} onClick={this.openStatusFlyout}>
+          <Trans i18nKey="rules.flyouts.disable">Disable</Trans>
+        </Btn>,
+      enable:
+        <Btn key="enable" className="rule-status-btn enabled" svg={svgs.enableToggle} onClick={this.openStatusFlyout}>
+          <Trans i18nKey="rules.flyouts.enable">Enable</Trans>
+        </Btn>,
+      changeStatus:
+        <Btn key="changeStatus" className="rule-status-btn" svg={svgs.changeStatus} onClick={this.openStatusFlyout}>
+          <Trans i18nKey="rules.flyouts.changeStatus">Change status</Trans>
+        </Btn>,
+      edit:
+        <Btn key="edit" svg={svgs.edit} onClick={this.openEditRuleFlyout}>
+          {props.t('rules.flyouts.edit')}
```

Comment:
The others use Trans but this one doesn't.  Should be consistent within the same file... one way or another.